elasticgraph-datastore_core/spec/integration/elastic_graph/datastore_core/index_definition/rollover_index_template_spec.rb

```diff
@@ -556,11 +547,7 @@ def build_datastore_core_for(rollover_frequency:, index_config: config_index_def
 
         def derive_index_from_template(record, datastore_core)
           indexer = build_indexer(datastore_core: datastore_core)
-          # The tests above use a slightly different schema definition than the main test schema definition,
-          # and as a result calls to `validate_mapping_completeness_of!` will fail. This is intentional, and
-          # we want to allow it, so here we stub it to be a no-op.
-          allow(indexer.datastore_router).to receive(:validate_mapping_completeness_of!)
-
+          # The tests above use a slightly different schema definition than the main test schema definition.
```

Comment:
```suggestion
```

/nit I don't think this is pertinent anymore. (It's not clear why this matters for the `index_into` call happening on the next line that it's appears to be explaining.).